examples/ads.amp.html

```diff
@@ -1024,14 +1025,24 @@ <h2 id="purch">Purch 300x250</h2>
     <div fallback></div>
   </amp-ad>
 
+  <h2 id="capirs">Rambler&Co</h2>
+  <amp-ad width="500" height="250"
```

Comment:
The thing is that the size of the advertisement is not known until it is loaded.
In real placements it is expected to customize the advertising and the place in such a way that they sizes will be equal or dynamic.
In the example, I can specify the size as auto. This will work, but I don't know will it be better or not.